packages/codemods/src/codeMods/utilities/transforms.ts

```diff
@@ -57,21 +126,29 @@ export function enumTransform(map: ValueMap<string>): PropTransform {
     if (!map) {
       return;
     }
-    switch (element.getKind()) {
-      case SyntaxKind.JsxExpression:
-        {
-          const toChange = Maybe(element.getFirstChildByKind(SyntaxKind.PropertyAccessExpression));
-          if (toChange.just) {
-            const oldText = toChange.value.getText();
-            toChange.value.replaceWithText(map[oldText]);
-          }
-        }
-        break;
-      case SyntaxKind.JsxOpeningElement:
-      case SyntaxKind.JsxSelfClosingElement: {
-        renamePropInSpread(element as JsxOpeningElement | JsxSelfClosingElement, toRename, replacementName, map);
-        break;
+    if (elementNotInSpread(element)) {
+      const toChange = getValueToChange(element as JsxExpression);
+      if (toChange) {
+        const oldText = toChange.getText();
+        toChange.replaceWithText(map[oldText]);
       }
+    } else {
+      renamePropInSpread(element as JsxOpeningElement | JsxSelfClosingElement, toRename, replacementName, map);
     }
   };
 }
+
+/* Returns whether the supplied ELEMENT contains a prop within a
+   spread attribute. This is done by detecting whether the supplied
+   ELEMENT is a JsxExpression, meaning that we have close access to
+   the prop, or if it's a Jsx opening element, meaning we can't access it. */
+function elementNotInSpread(element: JsxExpression | JsxOpeningElement | JsxSelfClosingElement): boolean {
+  return element.getKind() === SyntaxKind.JsxExpression;
+}
+
+/* Returns the piece of the AST that represents the value that the
+   developer can overwrite. */
+function getValueToChange(element: JsxExpression) {
+  const toChange = Maybe(element.getChildAtIndex(1)); // Child between {} operators.
```

Comment:
Since tochange isn't ever operated on it's okay to just do `return element.getChildAtIndex(1)`